pkg/policy/repository.go

```diff
@@ -603,3 +611,88 @@ func (p *Repository) GetPolicySnapshot() map[identity.NumericIdentity]SelectorPo
 
 	return p.policyCache.GetPolicySnapshot()
 }
+
+func RepositoryScriptCmds(p *Repository) map[string]script.Cmd {
+	return map[string]script.Cmd{
+		"policyrepo/list": script.Command(
+			script.CmdUsage{
+				Summary: "List all policies in the policy repository",
+			},
+			func(s *script.State, args ...string) (script.WaitFunc, error) {
+				return func(s *script.State) (stdout string, stderr string, err error) {
+					policies := p.GetRulesList()
+					return string(policies.Policy), "", nil
+				}, nil
+			},
+		),
+		"policyrepo/selectorcache": script.Command(
+			script.CmdUsage{
+				Summary: "List all policies in the policy repository",
```

Comment:
Summary is a copy from `policyrepo/list`, needs to be changed to describe `policyrepo/selectorcache`.